docs/user/FlowVariables.md

```diff
@@ -118,42 +121,62 @@ Note:
+| `FLOORPLAN_DEF`       | Use the DEF file to initialize floorplan.                                                                                                                                        |
 | `PLACE_SITE`          | Placement site for core cells defined in the technology LEF file.                                                                                                                |
 | `TAPCELL_TCL`         | Path to Endcap and Welltie cells file.                                                                                                                                           |
+| `RTLMP_FLOW`          | Enable the Hierarchical RTLMP flow. By default it is disabled.                                                                                                                   |
+| `MACRO_PLACEMENT`     | Specifies the path of a file on how to place certain macros manually using read_macro_placement.                                                                                 |
+| `MACRO_PLACEMENT_TCL` | Specifies the path of a TCL file on how to place certain macros manually.                                                                                                        |
 | `MACRO_PLACE_HALO`    | horizontal/vertical halo around macros (microns).                                                                                                                                |
 | `MACRO_PLACE_CHANNEL` | horizontal/vertical channel width between macros (microns).                                                                                                                      |
+| `MACRO_BLOCKAGE_HALO` | Blockage width overridden from default calculation.                                                                                                                              |
 | `PDN_CFG`             | File path which has a set of power grid policies used by `pdn` to be applied to the design, such as layers to use, stripe width and spacing to generate the actual metal straps. |
 | `MAKE_TRACKS`         | Tcl file that defines add routing tracks to a floorplan.                                                                                                                         |
 | `IO_PLACER_H`         | The metal layer on which to place the I/O pins horizontally (top and bottom of the die).                                                                                         |
 | `IO_PLACER_V`         | The metal layer on which to place the I/O pins vertically (sides of the die).                                                                                                    |
+| `GALLERY_REPORT`      | Enables to dump layout in .def extension. Default disabled.                                                                                                                      |

```

Comment:
The gallery report is mostly for internal use (project reporting) so I wouldn't bother to document it for users.